migration/migrator/migrations/course/20231113195556_create_upduck_table.py

```diff
@@ -0,0 +1,53 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+
+    database.execute(
+    """
+        CREATE TABLE IF NOT EXISTS upducks_table (
```

Comment:
Typically we don't include "table" in our table names.  Perhaps `forum_upducks` would be more appropriate.